nep-17.mediawiki

```diff
@@ -1,7 +1,7 @@
 <pre>
   NEP: 17
   Title: Token Standard
-  Author: Erik Zhang <erik@neo.org>
+  Author: Erik Zhang <erik@neo.org>, Vitor Nazário Coelho <vncoelho@gmail.com>
```

Comment:
Also add @belane he is the author of https://github.com/neo-project/proposals/issues/108

Reply:
Good @shargon.

I will add in this PR.
Can add yours as well?

Reply:
Added.